src/main/java/com/google/sps/data/NSWGovAQDataPoint.java

```diff
@@ -5,7 +5,8 @@
 
 import com.google.gson.annotations.SerializedName;
 
-/** A class, classified as NSWGovAQDataPoint */
+/** A class, classified as NSWGovAQDataPoint 
```

Comment:
When using JavaDoc style comments:
https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line
```
/**
 * Text goes here.
 * And here
 */
```
Top line is only `/**`, following lines have the `*` aligned under the first `*` above, final line is `*/`.